src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,57 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final String dataSeparator = " ";
+    private static final DateTimeFormatter dateFormat = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final int DATE_FROM_DATA_INDEX = 0;
+    private static final int EMPLOYEE_NAME_INDEX = 1;
+    private static final int HOURS_WORKED_PER_DAY_INDEX = 2;
+    private static final int INCOME_PER_HOUR_INDEX = 3;
+    private String dateFromData;
+    private String employeeName;
+    private int hoursWorkedPerDay;
+    private int incomePerHour;
```

Comment:
let's make these 4 variables local (move them to method)
this is not a very good practice to store state in services